pandas-stubs/_typing.pyi

```diff
@@ -437,7 +437,7 @@ class SequenceNotStr(Protocol[_T_co]):
     def __contains__(self, value: object, /) -> bool: ...
     def __len__(self) -> int: ...
     def __iter__(self) -> Iterator[_T_co]: ...
-    def index(self, value: Any, /, start: int = 0, stop: int = ...) -> int: ...
+    def index(self, value: Any, start: int = 0, stop: int = ..., /) -> int: ...
```

Comment:
https://github.com/microsoft/pyright/issues/6864

Reply:
Thank you! Can you please also replace `= 0` with `= ...`.

The CI caches the last installed dependencies: you will need to change pyproject.toml to get the latest pyright version.